### DISCUSSION TITLE
ESP32S3-N8R2 MicroPython

### DISCUSSION THREAD

**Discussion by Hamo-creator** [micropython/micropython#10654] [ESP32]
Hallo,
i have bought recently a new esp borad (ESP32S3 DEVKITC-1-N8R2) but i am having a problem with flash my code on it.
I am using Thonny as IDE on Windows.
I installed the MicroPython file on the board: GENERIC_S3-20220117-v1.18 (1).bin
I serached on the Internet for a solution but i found a lot of tips for Linux or Mos.
I tried to intall esptool using pip install esptool but it doesn't work.
Can someone help me and give an idea?

Thanks in forward

**Comment by davefes** [comment]
Does this help?

https://github.com/orgs/micropython/discussions/10187

**Reply by Hamo-creator** [reply]
Thank you.
It was helpfull
